### DISCUSSION TITLE
Anyone else having a choppy RTSP Stream?

### DISCUSSION THREAD

**Discussion by MichaelDiRienzo219** [mrlt8/docker-wyze-bridge#858] [General]
I've been using the docker wyze bridge for months without issues but starting yesterday I noticed my streams were having trouble connecting or when they did connect, about every 2 seconds the frame would update (not really a live feed like it's supposed to be). 

I am using Wyze Cam v3.

**Comment by mrlt8** [comment]
Was this after updating the bridge? Could it be some interference on the wifi?

**Reply by MichaelDiRienzo219** [reply]
I did only notice after I updated the bridge but I had restored a backup of the previous version and it was still happening. Thinking I might uninstall and reinstall tonight to see what happens.

**Reply by MichaelDiRienzo219** [reply]
if it was interference on the wifi, how would I diagnose that? (I haven't setup any new devices)

**Comment by spuggy1968** [comment]
I don't know if it has anything to do with this docker, I have noticed that Wyze directly app as been showing stream issues at times.

**Reply by MichaelDiRienzo219** [reply]
yeah, my Wyze app stream is totally fine which is the annoying part. I'm using the docker bridge to push the stream to Scrypted then Homekit.  The issue stems back all the way to the docker wyze bridge which is where im assuming its starting since the stream is looking good on the Wyze app.

**Comment by MichaelDiRienzo219** [comment]
Just updated to 2.3.4 and it seems to have fixed the issue. Thanks for your help!
